articles/fuzzy-logic-operations/index.md

```diff
@@ -0,0 +1,138 @@
+Fuzzy logic is an approach to solving problems that take into consideration the degrees of truth. Fuzzy Systems as a subject developed to model the uncertainty and vagueness present in the human thought process. 
+
+In this article, we will understand the following concepts.
+
+### Prerequisites
+
+This article on fuzzy logic is an excellent introduction to the basics of crisp and fuzzy sets. Moreover, it introduces the representation of fuzzy sets and their applications. 
+
+### Table of contents
+
+- Degree of membership function
+- Types of membership functions
+- Operations in fuzzy systems
+- Types of operations
+
+### Degree of membership function
+
+A fuzzy set is denoted as a tuple that includes the element and a value indicating its degree of membership to the universal set. Mathematically, it is represented as:
```

Comment:
It would be great if you could define what the degree of membership is, in a line or two